deploy/LOCAL_INSTALL.md

```diff
@@ -177,14 +177,49 @@ annotations, but you will need to re-load previously loaded projects to get the
    
    ``` 
 
-#### Adding a loaded dataset to a seqr project
+### Adding a loaded dataset to a seqr project
 
 After the dataset is loaded into elasticsearch, it can be added to your seqr project with these steps:
 
 1. Go to the project page
 1. Click on Edit Datasets
 1. Enter the elasticsearch index name (the `$INDEX_NAME` argument you provided at loading time), and submit the form.
 
-#### Enable read viewing in the browser (optional)
+### Enable read viewing in the browser (optional)
 
 To make .bam/.cram files viewable in the browser through igv.js, see **[ReadViz Setup Instructions](https://github.com/broadinstitute/seqr/blob/master/deploy/READVIZ_SETUP.md)**      
+
+### Loading RNASeq datasets
+
+Currently, seqr has a preliminary integration for RNA data, which requires the use of publicly available 
+pipelines run outside of the seqr platform. After these pipelines are run, the output must be annotated with metadata 
+from seqr to ensure samples are properly associated with the correct seqr families. After calling is completed, it can
```

Comment:
Are `the output must be annotated with metadata from seqr` and `After calling is completed` the same process?  Or is the seqr metadata annotating something that happens internally upon data load?